Automattic/wp-calypso
# Remove all instances of $gray-darken-10, $gray-darken-20, $gray-darken-30 by replacing them with the corresponding CSS variables

#### Changes proposed in this Pull Request

* Remove all instances of $gray-darken-10
* Remove all instances of $gray-darken-20
* Remove all instances of $gray-darken-30

#### Testing instructions

* Navigate to the calypso.live link and check if affected pages and components match the current state on the staging/production environments.
